src/genegraph/util.clj

```diff
@@ -0,0 +1,9 @@
+(ns genegraph.util
+  (:import (java.io ByteArrayInputStream)))
+
+(defn str->bytestream
+  "If s is a string, return an input stream of its contents, else return s"
+  [s]
+  (if (string? s)
+    (-> s .getBytes ByteArrayInputStream.)
+    s))
```

Comment:
Looks like this should be moved into `genegraph.database.load/read-rdf`.

Reply:
I was thinking of putting this inside `genegraph.database.load/read-rdf`, because most calls to `read-rdf` are passing a string that gets its contents converted to an InputStream by the caller, but there are a couple places where a string is passed to `read-rdf` and then passed to Jena as a string, which gets interpreted by Jena as a file URI. We'd need to distinguish between passing a string with the contents of the string being the document, and passing a string with the contents being a file URI pointing to the document.